package.json

```diff
@@ -14,6 +14,8 @@
         "archiver": "^3.1.1",
         "async": "^3.1.0",
         "bluebird": "^3.7.1",
+        "canvas": "^2.7.0",
```

Comment:
where is this lib used? I don't see any usages, whereas it adds around 40Mb of binaries as dependencies. Are you sure it's needed?